app/src/main/res/layout/content_layuot.xml

```diff
@@ -0,0 +1,263 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/content_view_wrapper"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/main"
+    android:orientation="vertical">
+
+    <ScrollView
```

Comment:
(!) I strongly recommend to rewrite this from scratch using `RecyclerView`.

- First, you are not gonna need `RelativeLayout` here, looks like you can use `FrameLayout`

- Second, now the layout itself is completely unreadable. Consider using `RecyclerView` instead of `ScrollView`, and just fill the recycler with different view holders.